server/api/store.go

```diff
@@ -609,7 +586,7 @@ func (filter *storeStateFilter) filter(stores []*metapb.Store) []*metapb.Store {
 
 func getStoreLimitType(input map[string]interface{}) (storelimit.Type, error) {
 	typeNameIface, ok := input["type"]
-	typeValue := storelimit.RegionAdd
+	typeValue := storelimit.AddPeer
```

Comment:
Will we consider to `RemovePeer`?

Reply:
The default output is `AddPeer`. It is the same as before.